include/service/entities/site_query_service.js

```diff
@@ -260,13 +260,20 @@ module.exports = function SiteQueryServiceModule(pb) {
    * @param {Function} cb
    */
   SiteQueryService.prototype.getCollections = function (cb) {
-    this.listCollections({ name: { $nin: ["system.indexes", "system.namespaces"] }}, function(err, items) {
-      if(pb.util.isError(err)) {
-        pb.log.error(err);
-      }
-      cb(err, items);
-    });
-  };
+      this.listCollections({}, function(err, items) {
```

Comment:
It will probably be more efficient to hand off the filtering to the DB.  The object you pass as the first parameter is the filter object.  You could specify that the name should not match a regular expression and filter it that way rather than writing additional code.